fs/fs.go

```diff
@@ -165,11 +171,19 @@ func (self *RealFsInfo) addDockerImagesLabel(context Context) {
 		dockerPaths := getDockerImagePaths(context)
 
 		for src, p := range self.partitions {
-			self.updateDockerImagesPath(src, p.mountpoint, dockerPaths)
+			self.updateContainerLabelImagesPath(LabelDockerImages, src, p.mountpoint, dockerPaths)
```

Comment:
It is kind of weird to make `addDockerImagesLabel` update labels for `rkt` as well. This logic might require a more thorough refactoring.